HouseRules_Core/README.md

```diff
@@ -58,10 +66,12 @@ loadRulesetsFromConfig = true
 A list of out-of-the-box Ruleset names can be found in the
 [HouseRules_Essentials readme](../HouseRules_Essentials/README.md).
 
-To use a built-in ruleset, ensure `loadFromConfig` is set to `false` and select the built-in ruleset to use by typing its name within the quotes.
-Alternatively, use empty quotes `""` to specify no ruleset should be used.
+The name of a default ruleset can be set in `defaultRuleset` - alternatively, use empty quotes `""` to specify no ruleset should be used.
+The `loadRulesetsFromConfig` flag can be set to `false` to prevent loading of JSON rulesets.
+
+## User-Defined JSON Rulesets
 
-Rulesets may also be defined as JSON files  `UserData/HouseRules/<ruleset_name>.json`. Here's a sample ruleset for beefing up the Sorcerer's Zap ability. 
+JSON rulesets can be defined within the Demeo game directory and saved in  `UserData/HouseRules/` as files named `<ruleset_name>.json`. The `HouseRules` folder may need to be manually created before use. Below is a sample ruleset for improving the life of the Sorcerer. It removes the casting cost on his Zap ability and gives it an extra point of damage.
```

Comment:
> folder may need to be manually created before use

Heads up: The mod will automatically create the directory if missing.

Reply:
Ah cool... I knew it created it if I told it to write out the rulesets but wasn't sure if it created it otherwise. I've updated the docs accordingly now.